lib/timeline/Timeline.js

```diff
@@ -428,32 +428,38 @@ Timeline.prototype.focus = function(id, options) {
       }
     };
     
-    // Perform one last check at the end to make sure the final vertical
-    // position is correct
-    var finalVerticalCallback = function() {
+    // Enforces the final vertical scroll position
+    var setFinalVerticalPosition = function() {
       var finalVerticalScroll = getItemVerticalScroll(me, item);
 
-      if(finalVerticalScroll.shouldScroll && finalVerticalScroll.itemTop != initialVerticalScroll.itemTop) {              
+      if (finalVerticalScroll.shouldScroll && finalVerticalScroll.itemTop != initialVerticalScroll.itemTop) {
         me._setScrollTop(-finalVerticalScroll.scrollOffset);
-        me._redraw();        
+        me._redraw();
       }
     };
 
+    // Perform one last check at the end to make sure the final vertical
+    // position is correct
+    var finalVerticalCallback = function() {
+      // Double check we ended at the proper scroll position
+      setFinalVerticalPosition();
+
+      // Let the redraw settle and finalize the position.      
+      setTimeout(setFinalVerticalPosition, 100);
```

Comment:
Try `requestFrameAnimation` instead of `setTimeout`

Reply:
I don't think that will accomplish what I'm trying to do here. The core issue that I noticed is the timeline resizes groups as you scroll horizontally, and if that resize happens just after the vertical scroll finishes then the "final" position the user sees is not the item that they asked to have focused. I don't have an event I can hook into for that at the moment, so the best I've got is  a small delay after the vertical scroll has finished to essentially do a "double check". `requestAnimationFrame` would likely call this too soon, which is why I chose `setTimeout`. If you have a suggestion for an event or something else I could check that would be more reliable I'll gladly use that.

Reply:
mmm Not really. I'll approve it for now. I'l think about it more though.

Reply:
I'm wanting to try and do some tracing of the various events that are being emitted so I can understand where and when the recalculations come from post-focus. If I find something l'll let you know.